tests/nested/manual/core20-install-device-file-install-ubuntu-save-via-hook/task.yaml

```diff
@@ -0,0 +1,143 @@
+summary: Verify that ubuntu-save is available during install-device hook with specific device
+
+details: |
+    This test checks that the workaround put in place for a specific device
+    continues to work. This specific device needs to install configuration files
+    onto the run system ubuntu-data on UC20 during install mode to avoid an
+    additional reboot during the first boot process.
+
+systems: [ubuntu-20.04-64, ubuntu-22.04-64]
+
+environment:
+    # use snapd from the spread run so that we have testkeys trusted in the
+    # snapd run
+    NESTED_BUILD_SNAPD_FROM_CURRENT: true
+
+    # use secure boot and TPM to enable encryption
+    NESTED_ENABLE_TPM: true
+    NESTED_ENABLE_SECURE_BOOT: true
+
+    # don't use cloud-init it doesn't work with grade secured
+    NESTED_USE_CLOUD_INIT: false
+
+    # sign all the snaps we build for the image with fakestore
+    NESTED_SIGN_SNAPS_FAKESTORE: true
+
+    # use the testrootorg auto-import assertion
+    NESTED_CUSTOM_AUTO_IMPORT_ASSERTION: $TESTSLIB/assertions/developer1-{VERSION}-auto-import.assert
+    NESTED_CUSTOM_MODEL: $TESTSLIB/assertions/developer1-{VERSION}-secured.model
+
+    # for the fake store
+    NESTED_FAKESTORE_BLOB_DIR: $(pwd)/fake-store-blobdir
+    NESTED_UBUNTU_IMAGE_SNAPPY_FORCE_SAS_URL: http://localhost:11028
+
+    # unset this otherwise ubuntu-image complains about overriding the channel for
+    # a model with grade higher than dangerous when building the image
+    NESTED_CORE_CHANNEL: ""
+
+prepare: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+
+    # install pre-reqs which we need to adjust various bits
+    snap install jq remarshal
```

Comment:
I don't see `marshal` being used in the tests; what is the command that it provides?

Reply:
It provides the command `yaml2json` which is used during prepare in some of the test scripts. It's not directly used in the test yaml, but it seems to be neccessary to install

Reply:
I see, thanks!